src/main/java/core/basesyntax/service/operation/ReturnOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.operation;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
The class name `ReturnOperation` could be more descriptive. Consider renaming it to reflect that it handles the 'return' operation for fruits, for example, `FruitReturnOperationHandler`.